### PR TITLE
Option to disable Encrypt-then-MAC in s_client

### PR DESCRIPTION
**Pull request by beldmit**
##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: doc/man1/openssl-s_client.pod.in [hunk 1]
```diff
@@ -448,6 +449,10 @@ Enable send/receive timeout on DTLS connections.
 
 Set MTU of the link layer to the specified size.
 
+=item B<-no_etm>
+
+Disable Encrypt-then-MAC negotiation
```
Comment (paulidale) [inline doc/man1/openssl-s_client.pod.in:454]: Full stop at the end.

**Review by paulidale — APPROVED** [review]

File: doc/man1/openssl-s_client.pod.in (hunk 1 above)
Comment (beldmit) [reply doc/man1/openssl-s_client.pod.in]: Fixed, thanks!

**Review by paulidale — APPROVED** [review]

**Comment by tomato42** [comment]
why just on the client side?

**Comment by beldmit** [comment]
> why just on the client side?

Fixed. @paulidale, would you mind to reapprove?

**Review by paulidale — DISMISSED** [review]

**Comment by kaduk** [comment]
> why just on the client side?

There are server implementations that choke badly if you send ETM as a client. A client that will behave badly with ETM isn't going to send it in the first place, so a properly behaving server like `s_server` doesn't need a particular option to disable it.

**Comment by tomato42** [comment]
@kaduk we're introducing it in the context of #15706 
Also, having the option disable negotiation of EtM on server side when we're testing interoperability with client that can't disable it, at least allows us to still test it.

**Comment by kaduk** [comment]
@tomato42 you can already use a config file with `-EncryptThenMac` (and could for `s_client` as well) if you really need to.  What Dmitry is adding here is a convenience option, and I'm mostly only seeing the need for the convenience option on the client side, if at all.

**Comment by beldmit** [comment]
Reimplemented via OPT_S, thanks @tomato42 
@paulidale please reapprove, hopefully the last time

**Review by t8m — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Comment by beldmit** [comment]
Merged after resolving small conflicts.

**Pull request closed without merging**